.yarnrc.yml

```diff
@@ -0,0 +1,7 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-engines.cjs
```

Comment:
one is local and one is remote, weird :) but I assume it is something yarn do :D

Reply:
Yep, the first is the local path and the second is where to download it if the file is not present. This should prevent failing when the file is missing and instead it will try to download it before it fails.

See: https://github.com/yarnpkg/berry/issues/4479